asciidoc-confluence-publisher-doc/etc/docs/00-index.adoc

```diff
@@ -123,6 +123,14 @@ The Confluence Publisher is configured with the help of a Maven plugin. A typica
 | The password of the user to use for publishing.
 | mandatory
 
+| strategy
```

Comment:
I would opt for a more specific name in all client integrations (Maven, Docker, CLI), i.e. `publishStrategy` - in order to avoid potential name clashes in the future (with other strategies yet to be added).